modules/ROOT/examples/live-demos/custom-toolbar-menu-button-search/index.js

```diff
@@ -0,0 +1,53 @@
+tinymce.init({
+  selector: 'textarea#custom-toolbar-menu-button-search',
+  height: 500,
+  toolbar: 'mybutton',
+
+  setup: (editor) => {
+    /* Menu items are recreated when the menu is closed and opened, so we need
+       a variable to store the toggle menu item state. */
+    let toggleState = false;
+
+    /* example, adding a toolbar menu button */
+    editor.ui.registry.addMenuButton('mybutton', {
+      text: 'My searchable button',
+      search: {
+        placeholder: 'Type...'
+      },
+      fetch: (callback, fetchContext) => {
+        if (fetchContext.pattern.length > 0) {
+          callback([
+            {
+              type: 'menuitem',
+              text: `You searched for: "${fetchContext.pattern}"`,
+              onAction: () => editor.insertContent(`<strong>Inserted selected search result</strong>`)
+            }
+          ])
```

Comment:
```suggestion
          ]);
```

Missing a semi colon :)